css/style.css

```diff
@@ -283,6 +315,132 @@ header {
   justify-content: center;
 }
 
+/* popup */
+
+.popup-cover {
+  position: fixed;
+  display: none;
+  z-index: 200;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  backdrop-filter: blur(6px);
+  background-color: rgb(193, 199, 208, 0.4);
+}
+
+.modal {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  position: relative;
+  width: auto;
+  background-color: #fff;
+  margin: 16px;
+  border-radius: 10px;
+  padding: 10% 16px;
+}
+
+.modal .close-modal {
+  position: absolute;
+  top: 6px;
+  right: 2%;
+}
+
+.modal .close-modal .bi-x {
+  font-size: 34px;
+  color: #67798e;
+  font-weight: 700;
+}
+
+.modal .modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  align-content: center;
+  padding: 0;
+  margin-bottom: 20px;
+  width: 100%;
+  border: none;
+}
+
+.modal .project-title {
+  font-size: 32px;
+  font-weight: bold;
+  line-height: 44px;
+  color: #172b4d;
+  text-transform: capitalize;
+}
+
+.project-client {
+  font-size: 13px;
+  margin: 0;
+  padding: 0;
+  font-weight: 600;
+  line-height: 16px;
+  color: #344563;
+  text-transform: uppercase;
+}
+
+.modal .modal-header .role-year-ul {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  align-content: center;
+  width: 70%;
+}
+
+.project-role,
+.project-year {
+  color: #7a869a;
+  font-size: 13px;
+  line-height: 16px;
+  font-weight: 600;
+}
+
+.modal .modal-header .role-year-ul .project-year {
+  margin-left: 32px;
+}
+
+.modal .featured-image-container {
+  align-self: stretch;
+  max-height: 50%;
```

Comment:
- [ ] I think it would be nice to have designs of the images match with the figma design given, Currently, the max-height set here is causing images on the pop-up to only stretch horizontally as shown below:
#### Your project
![image](https://user-images.githubusercontent.com/61377322/146309762-15709d54-6cf7-4e56-a433-6d0b2a1ab694.png)

#### Figma design
![image](https://user-images.githubusercontent.com/61377322/146309774-2451ddb9-d32e-4fd4-a5b9-52cce4b3c41d.png)

Reply:
Hello, 
Your feedback is fixed.
Thank you.